frontend/src/index.js

```diff
@@ -4,12 +4,7 @@ import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
```

Comment:
Are you sure we don't need a StrictMode?
I remember reading about it somewhere that we should use it

Reply:
Your call though, I'm not entirely sure :p

Reply:
Yea.. The `navigate` to other pages was not working with strict mode. That's why I removed it. It wont create any issues.. ig

Reply:
Cool, taking @bodhisha's word on it. XD

Reply:
Haha.. Let's see.. 😛